cmd/state-installer/cmd.go

```diff
@@ -300,23 +305,23 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 		return postInstallEvents(out, cfg, an, params, isUpdate)
 	}
 
-	// Check if state tool already installed
-	if !params.force && stateToolInstalled {
-		logging.Debug("Cancelling out because State Tool is already installed")
-		out.Print(fmt.Sprintf("State Tool Package Manager is already installed at [NOTICE]%s[/RESET]. To reinstall use the [ACTIONABLE]--force[/RESET] flag.", installPath))
-		an.Event(AnalyticsFunnelCat, "already-installed")
-		return postInstallEvents(out, cfg, an, params, true)
-	}
+	storeInstallSource(params.sourceInstaller)
 
-	// If no sourcePath was provided then we still need to download the source files, and defer the actual
-	// installation to the installer contained within the source file
-	return installFromRemoteSource(out, cfg, an, args, params)
+	return locale.NewError("err_install_source_path_not_provided", "sourcePath was not provided by install.sh or install.ps1")
```

Comment:
```suggestion
	return locale.NewError("err_install_source_path_not_provided", "Installer was called without an installation payload. Please make sure you're using the install.sh or install.ps1 scripts.")
```

Localized errors are user facing.

Reply:
I was under the impression that this could only happen if we (the developers) miscoded something, so this error message was primarily for us. I can see that it's possible a user might attempt to invoke `state-installer` directly, and would thus run into this. Thanks for pointing it out.